pkg/securitypolicy/securitypolicy.go

```diff
@@ -86,6 +88,18 @@ func NewOpenDoorPolicy() *SecurityPolicy {
 	}
 }
 
+// NewSecurityPolicyDigest converts base64policy string into JSON and computes
+// sha256-digest on the JSON string.
```

Comment:
I would mention this returns a base64 encoded string. From the name I expected it to return `[]byte`.